tests/unit/rql.test.ts

```diff
@@ -308,17 +316,20 @@ describe('rql string builder', () => {
   });
 
   it('Should globally override the double encoding of RQL values', () => {
-    rqlBuilder.doubleEncodeValues = false;
+    rqlBuilder.doubleEncodeValues = true;
 
     const firstName = '~> Tester <~';
+    const doubleEncodedFirstName = '%257E%253E%2520Tester%2520%253C%257E';
+
     const lastName = '<! McTestFace !>';
+    const doubleEncodedLastName = '%253C%2521%2520McTestFace%2520%2521%253E';
 
     // As the global setting is false the rql builder should not double encode the values
     const firstRql = rqlBuilder().in('firstNames', [firstName]).build();
-    expect(firstRql).toStrictEqual(`?in(firstNames,${firstName})`);
+    expect(firstRql).toStrictEqual(`?in(firstNames,${doubleEncodedFirstName})`);
 
     const secondRql = rqlBuilder().out('lastNames', [lastName]).build();
-    expect(secondRql).toStrictEqual(`?out(lastNames,${lastName})`);
+    expect(secondRql).toStrictEqual(`?out(lastNames,${doubleEncodedLastName})`);
 
     // Reset the default global value
     rqlBuilder.doubleEncodeValues = true;
```

Comment:
Does this make sense at the end of the test? I believe we can remove it. There is another one exactly the same.

Reply:
Correct, i missed them.